back/scripts/enrichment/communities_enricher.py

```diff
@@ -16,10 +19,49 @@ def get_dataset_name(cls) -> str:
 
     @classmethod
     def get_input_paths(cls, main_config: dict) -> typing.List[Path]:
-        return [CommunitiesSelector.get_output_path(main_config)]
+        return [
+            CommunitiesSelector.get_output_path(main_config),
+            BaremeEnricher.get_output_path(main_config),
+        ]
 
     @classmethod
     def _clean_and_enrich(cls, inputs: typing.List[pl.DataFrame]) -> pl.DataFrame:
-        communities, *_ = inputs
-        # Data analysts, please add your code here!
-        return communities
+        communities, bareme = inputs
+
```

Comment:
```suggestion
        if len(inputs) != 2:
           raise ValueError("Expected exactly two input DataFrames: communities and bareme.")
        communities, bareme = inputs
```

Ajoutons un petit contrôle des données en entrée ;)